.github/workflows/oft.yaml

```diff
@@ -11,12 +11,18 @@ jobs:
     runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@v4
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.12.0
+        with:
+          python-version: "3.10"
+          poetry-version: '1.8.2'
```

Comment:
do you use locally still poetry 1.8.2? Please, update at some point and then also update this version. Is not needed now, but should happen at some point, because other projects already updated.

Reply:
Will do so...if ITDE updates Poetry, I will be forced to update....